docs/SUMMARY.md

```diff
@@ -32,6 +32,7 @@
 * [Authenticating Users via LDAP](Authenticating-Users-via-LDAP.md)
 * [Authenticating and Authorizing Users via Keycloak](Authenticating-and-Authorizing-Users-via-keycloak.md)
 * [Authenticating Users via Tokens](Authenticating-Users-via-Tokens.md)
+* [OncoKB Data Access](OncoKB-Data-Access.md)
```

Comment:
I'm not sure if auth is the best place to put this. It's more like an integration and has little to do with cBioPortal auth. Maybe we should add it to `2.3 Customization`. That's where we for instance mention CIVIC and others: https://docs.cbioportal.org/2.3-customization/portal.properties-reference. 

It does feel slightly different from customization in that it's more `integration with other web services`. Could also re-organize customization a bit and create new section `2.4 Integration with other web services`. Matchminer and CIVIC could maybe go there as well